xml/System.Security.Cryptography.Pkcs/SignedCms.xml

```diff
@@ -535,51 +535,26 @@
-  
--   The certificate is within its validity period.  
-  
--   The certificate has no key usage or a key usage that supports digital signatures or nonrepudiation.  
-  
- Signer certificates are chosen from the My store.  
-  
+
+This method succeeds if <xref:System.Security.Cryptography.Pkcs.SubjectIdentifierType.NoSignature?displayProperty=nameWithType> was provided as the `signerIdentifierType` argument of one of the <xref:System.Security.Cryptography.Pkcs.SignedCms> constructor overloads. Otherwise, it throws an exception.
+
 ## .NET Framework Security  
- The following permissions are required to display the user interface:  
+ 
+The following permissions are required to access the signature key:  
   
--   <xref:System.Security.Permissions.UIPermissionWindow.SafeTopLevelWindows?displayProperty=nameWithType>  
-  
- The following permissions are required to access the signature key:  
-  
--   <xref:System.Security.Permissions.KeyContainerPermissionFlags.Open?displayProperty=nameWithType>  
-  
--   <xref:System.Security.Permissions.KeyContainerPermissionFlags.Sign?displayProperty=nameWithType>  
-  
-   
-  
-## Examples  
- The following example shows the steps to compute a signature on a <xref:System.Security.Cryptography.Pkcs.SignedCms> message with message content that is not detached. In this case, the message content is included in the <xref:System.Security.Cryptography.Pkcs.SignedCms> message.  
-  
- [!code-csharp[mpkcs_SignedCMS#3](~/samples/snippets/csharp/VS_Snippets_WindowsServer/mpkcs_SignedCMS/cs/Program.cs#3)]
- [!code-vb[mpkcs_SignedCMS#3](~/samples/snippets/visualbasic/VS_Snippets_WindowsServer/mpkcs_SignedCMS/vb/Program.vb#3)]  
-  
- The following example shows the steps to compute a signature on a <xref:System.Security.Cryptography.Pkcs.SignedCms> message with message content that is detached. In this case, the message content must be verified independently of the <xref:System.Security.Cryptography.Pkcs.SignedCms> message.  
-  
- [!code-csharp[mpkcs_SignedCMS#4](~/samples/snippets/csharp/VS_Snippets_WindowsServer/mpkcs_SignedCMS/cs/Program.cs#4)]
- [!code-vb[mpkcs_SignedCMS#4](~/samples/snippets/visualbasic/VS_Snippets_WindowsServer/mpkcs_SignedCMS/vb/Program.vb#4)]  
+- <xref:System.Security.Permissions.KeyContainerPermissionFlags.Open?displayProperty=nameWithType>  
   
+- <xref:System.Security.Permissions.KeyContainerPermissionFlags.Sign?displayProperty=nameWithType>  
+
  ]]></format>
         </remarks>
-        <exception cref="T:System.ArgumentNullException">A null reference was passed to a method that does not accept it as a valid argument.</exception>
-        <exception cref="T:System.Security.Cryptography.CryptographicException">A cryptographic operation could not be completed.</exception>
+        <exception cref="T:System.InvalidOperationException">.NET Framework (all versions) and .NET Core (version 2.2 and later): The recipient certificate is not specified.</exception>
```

Comment:
The .NET Core change isn't part of 2.2, but it will be for 3.0.  (Sorry, should have noticed that last pass)

Reply:
If it will be for 3.0, perhaps we should remove it until an early release version of 3.0 is available. @bartonjs what do you think?